### ISSUE TITLE
[Feature] Add prompt editor

### ISSUE THREAD

**Issue by BrianBorni** [ArtVentureX/sd-webui-agent-scheduler#57]
When you make a mistake and you have a lot of prompts with that mistake in the queue, is possible to make a feature to can edit the prompts that are in the queue?

**Comment by artventuredev** [comment]
Thanks for the suggestion. This feature is already in the backlog.

**Comment by PaperOrb** [comment]
This feature will singlehandedly take this to the next level for me, so I'm glad to hear it wasn't just me.

**Comment by aleph23** [comment]
Yeah, if this were made a reality, I'd stop disabling this extension.

**Comment by artventuredev** [comment]
My bad, this feature is already available for a while. You can double click a task and update some basic parameters.
 
![Screenshot 2023-08-03 at 16 35 53](https://github.com/ArtVentureX/sd-webui-agent-scheduler/assets/133728487/b9d8d8a5-ba96-481e-8b87-6f56e37ff6c4)
